packages/SwingSet/src/lib/workerOptions.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @param {import("../types-external").KernelKeeper} kernelKeeper
+ * @param {import("../controller/bundle-handler").BundleHandler} bundleHandler
+ * @param {string} [managerType]
+ * @returns {Promise<import("../types-internal").WorkerOptions>}
+ */
+export async function makeWorkerOptions(
+  kernelKeeper,
+  bundleHandler,
+  managerType,
+) {
```

Comment:
Suggested signature refactoring: make `type` the first argument and drop `kernelKeeper` (moving use of it to call sites destructuring `managerType = kernelKeeper.getDefaultManagerType()` like they do with `reapInterval = kernelKeeper.getDefaultReapInterval()`).
```suggestion
/**
 * @param {string} managerType
 * @param {import("../controller/bundle-handler").BundleHandler} bundleHandler
 * @returns {Promise<import("../types-internal").WorkerOptions>}
 */
export async function makeWorkerOptions(managerType, bundleHandler) {
```